### PR TITLE
Add type None, to return no min or minutes

### PR DESCRIPTION
**Pull request by lukasoppermann**
"Fixes" #4.

### CODE REVIEW

File: src/Bookworm/Bookworm.php [hunk 1]
```diff
@@ -63,9 +64,10 @@ public static function estimate($text, $type = self::TYPE_SHORT)
         $minutes = $minutes > 1 ? $minutes : 1;
 
         // Get the proper textual representation
-        $textual = self::$translations[$type == self::TYPE_SHORT ? 'min' : 'minute'];
-
-        return $minutes . ' ' . $textual;
+        if($type !== self::TYPE_NONE){
```
Comment (jeroenvisser101) [inline src/Bookworm/Bookworm.php]: Can you apply spaces as defined in [PSR-2](http://www.php-fig.org/psr/psr-2/) here?

File: src/Bookworm/Bookworm.php [hunk 2]
```diff
@@ -63,9 +64,10 @@ public static function estimate($text, $type = self::TYPE_SHORT)
...
+            $textual = ' '.self::$translations[$type == self::TYPE_SHORT ? 'min' : 'minute'];
```
Comment (jeroenvisser101) [inline src/Bookworm/Bookworm.php]: Here as well :)

File: src/Bookworm/Bookworm.php [hunk 3]
```diff
@@ -63,9 +64,10 @@ public static function estimate($text, $type = self::TYPE_SHORT)
...
+        }
+        return $minutes . (isset($textual) ? $textual : '');
```
Comment (jeroenvisser101) [inline src/Bookworm/Bookworm.php:71]: and a newline before the return statement :)

**Comment by jeroenvisser101** [comment]
:+1: some coding standards, but that's pretty nice! :)

**Comment by lukasoppermann** [comment]
Should be fixed.

**Comment by jeroenvisser101** [comment]
Great! Thanks :+1:

**Pull request merged**